db/seeds.rb

```diff
@@ -2,10 +2,21 @@
 
 WasteExemptionsEngine::Engine.load_seed
 
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+def find_or_create_user(email, role)
+  User.find_or_create_by(email: email) do |user|
+    user.role = role
+    user.password = ENV["WEX_DEFAULT_PASSWORD"] || "Secret123"
```

Comment:
I've dropped the convention of prefixing environment variables with the service's acronym for WEX, in the hope that it'll make the code more reusable going forward.

Other than local development (where a .env file would be used anyway) we'll never have WCR and WEX running in the same instance, so there is no need to prefix variables to avoid duplication.

Reply:
Makes sense to me! Updated the env vars.